server/api/src/rsDataObjOpen.cpp

```diff
@@ -1019,7 +1021,7 @@ namespace
             }};
 
             if (const int unlock_ec = ill::unlock_and_publish(*rsComm, {replica, admin_op, 0}, ill::restore_status); unlock_ec < 0) {
-                irods::log(LOG_ERROR, fmt::format("[{}:{}] - failed to unlock data object [error_code={}]", __FUNCTION__, __LINE__, unlock_ec));
+                irods::log(LOG_NOTICE, fmt::format("[{}:{}] - failed to unlock data object [error_code={}]", __FUNCTION__, __LINE__, unlock_ec));
```

Comment:
what's the reasoning?   just more visibility?

Reply:
After seeing it pop up as a result of a more important overarching error, I didn't feel that it was at the level of ERROR, but perhaps WARNING would be more appropriate. I am willing to be convinced.

Reply:
so I guess the question is... does it require human eyeballs, that's ERROR...  If it could cause a problem, then WARNING seems right.   If it's merely informative... NOTICE seems good.

Reply:
Perhaps ERROR is the best category, then. I'll put it back